pkgs/development/python-modules/analytics-python/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, monotonic
+, backoff
+, python-dateutil
+}:
+
+buildPythonPackage rec {
+  pname = "analytics-python";
+  version = "1.4.0";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "plFBq25H2zlvW8Vwix25P/mpmILYH+gIIor9Xrtt/l8=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py \
+      --replace '"backoff==1.10.0"' '"backoff>=1.10.0,<3"'
+  '';
+
+  propagatedBuildInputs = [
+    monotonic
+    requests
+    backoff
+    python-dateutil
+  ];
+
+  # Almost all tests run against a hosted API, and the few that are mocked are hard to cherry-pick
+  doCheck = false;
+
+  pythonImportsCheck = [
+    "analytics"
+    "analytics.client"
+    "analytics.consumer"
+    "analytics.request"
+    "analytics.utils"
+    "analytics.version"
+  ];
+
+  meta = with lib; {
+    homepage = "https://segment.com/libraries/python";
+    description = "Hassle-free way to integrate analytics into any python application";
+    license = licenses.mit;
+    maintainers = with maintainers; [ pbsds ];
```

Comment:
I don't mind the maintainership, but is this perchance a package i packaged for an older version of gradio? A newer version is merged [here](https://github.com/NixOS/nixpkgs/pull/241558) which does not require analtics-python

Reply:
ah sry for pinging you - im trying to upstream nixified-ai and i just copied over the package from there
ill remove all maintainers from packages i add for now until i figure out most things and need help

Reply:
There is a high likelyhood this package is not needed, since it is a transitive dependency in gradio

Reply:
ok i just added all the packages that were in nixified ai but not in nixpkgs and ill review later what is needed
i still have to figure out some things about the nixified-ai flake so it might take some time